spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe SessionsController do
+  describe "GET new" do
+    it "renders new template for unauthenticated users" do
+      get :new
+      expect(response). to render_template :new
```

Comment:
This feels like you are testing Rails. `get :new` will render `:new` template by default. Probably
`expect(response).no_to redirect_to home_path` would be more accurate.